server/src/controllers/employer/getAllEmployers.js

```diff
@@ -16,12 +17,31 @@ const getAllEmployersRequestHandler = async (req, res) => {
       endDateAdded,
       ownerId,
       postalCode,
+      contactName,
     } = req.query;
 
     const query = {};
     if (employerName) {
       query.name = { [Op.like]: `%${employerName}%` };
     }
+
+    let employerContactIds = [];
+    if (contactName) {
+      const employerContacts = await EmployerContact.findAll({
+        where: {
+          name: { [Op.like]: `%${contactName}%` },
+        },
+      });
+      employerContactIds = employerContacts.map((contact) => contact.id);
+    }
+
+    logger.info(`Employer contact ids: ${employerContactIds}`);
+    logger.info(`Employer Name: ${contactName}`);
+
+    if (employerContactIds.length > 0) {
```

Comment:
I think its best to apply the filter whether or not there are any employers contacts with a given name (this indicates to a user when their search was unsuccessful). Of course this should only be done when contactName is set.